File: C++/1894. Find the Student that Will Replace the Chalk.cpp
class Solution {
public:
    int chalkReplacer(vector<int>& c, int k) {
        long long i,s=0;
        s=accumulate(c.begin(),c.end(),s);
        k=k%s;
        for(i=0;i<c.size();i++){
            if(k<0)
            return(i-1);
            k-=c[i];
        }
        return i-1;
    }
};

File: JAVA/1894. Find the Student that Will Replace the Chalk.java
class Solution {
    public int chalkReplacer(int[] c, int k) {
        int i;
        long s=0;
        for(i=0;i<c.length;i++){
            s+=c[i];
        }
        s=k%s;
        for(i=0;i<c.length;i++){
            if(s<0)
            return (i-1);
            s-=c[i];
        }
        return i-1;
    }
}